src/routes/group/Group.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import Card from 'components/Card/Card';
+
+import { groupOptions } from './groupOptions';
+
+import style from './Group.module.css';
+
+export default function Group() {
+  const navigate = useNavigate();
+
+  const handleNavigate = (i: number) => {
+    navigate(groupOptions[i].url, { replace: false });
+  };
+
+  return (
+    <div className={style.container}>
+      {groupOptions.map((option, i) => (
+        <Card
+          key={option.title}
+          className={style.card}
+        >
+          <div
+            className={style.option}
+            onClick={() => handleNavigate(i)}
+            role="button"
+            tabIndex={i}
```

Comment:
You are doing this to allow the user to click the card right? Consider using the Link component of ReactRouter instead. It makes more sense semantically (an anchor element to navigate to another page) and you won't have to worry about tab functionality